substratevm/src/com.oracle.svm.driver/src/com/oracle/svm/driver/APIOptionHandler.java

```diff
@@ -391,6 +393,10 @@ String translateOption(ArgumentQueue argQueue) {
                 builderOption += transformed.toString();
             }
 
+            if (option.launcherOption) {
+                nativeImage.bundleLauncherArgs.add(argQueue.peek());
```

Comment:
This needs some tweaking. In case `option.valueSeparator()` is `WHITESPACE_SEPARATOR` you have to also take the subsequent argument from `argQueue`. See  JavaDoc comment of `valueSeparator()` in `@APIOption` annotation.

Reply:
Addressed in commit https://github.com/oracle/graal/pull/6496/commits/939e77cd7c6373a307fb5f85a89ac3a5d9af9ee9